kubespawner/spawner.py

```diff
@@ -641,8 +642,8 @@ def _hub_connect_port_default(self):
            - `mountPath` The path on the container in which we want to mount the volume.
            - `name` The name of the volume we want to mount, as specified in the `volumes` config.
 
-        See `the Kubernetes documentation <https://kubernetes.io/docs/concepts/storage/volumes>`_ for more information on how
-        the `volumeMount` item works.
+        See `the Kubernetes documentation <https://kubernetes.io/docs/concepts/storage/volumes>`__
```

Comment:
Another double underscore, I did not know about this syntax, I've seen a single one being used within the z2jh docs throughout.

Reply:
double-underscore means "anonymous link" whereas single underscore is "named link" which creates a named ref other links can refer to. Having multiple named refs with the text "Kubernetes documentation" was causing a collision.

Reply:
Ah!